public/lang/de.json

```diff
@@ -1 +1,158 @@
+        "pair-devices-title": "Geräte dauerhaft koppeln",
+        "input-key-on-this-device": "Gebe diesen Schlüssel auf einem anderen Gerät ein",
+        "enter-key-from-another-device": "Gebe den Schlüssel von einem anderen Gerät hier ein.",
+        "pair": "Koppeln",
+        "cancel": "Abbrechen",
+        "edit-paired-devices-title": "Gekoppelte Geräte bearbeiten",
+        "paired-devices-wrapper_data-empty": "Keine gekoppelten Geräte.",
+        "close": "Schließen",
+        "accept": "Akzeptieren",
+        "decline": "Ablehnen",
+        "title-image": "Bild",
+        "title-file": "Datei",
+        "title-image-plural": "Bilder",
+        "title-file-plural": "Dateien",
+        "scan-qr-code": "oder scanne den QR-Code.",
+        "would-like-to-share": "möchte Folgendes teilen",
+        "send": "Senden",
+        "copy": "Kopieren",
+        "receive-text-title": "Textnachricht erhalten",
+        "file-other-description-image-plural": "und {{count}} andere Bilder",
+        "file-other-description-file-plural": "und {{count}} andere Dateien",
+        "activate-paste-mode-activate-paste-mode-shared-text": "Freigegebener Text",
+        "auto-accept-instructions-1": "Aktiviere",
+        "auto-accept": "auto-accept",
+        "auto-accept-instructions-2": "um automatisch alle Dateien von diesem Gerät zu akzeptieren.",
+        "has-sent": "hat Folgendes gesendet:",
+        "send-message-title": "Textnachricht senden",
+        "send-message-to": "Sende eine Textnachricht an",
+        "base64-tap-to-paste": "Tippe hier um {{type}} einzufügen",
+        "base64-paste-to-send": "Füge {{type}} hier zum Versenden ein",
+        "base64-text": "Text",
+        "base64-files": "Dateien",
+        "base64-processing": "Bearbeitung läuft…",
+        "file-other-description-image": "und 1 anderes Bild",
+        "file-other-description-file": "und 1 andere Datei",
+        "receive-title": "{{descriptor}} erhalten",
+        "download-again": "Erneut herunterladen",
+        "system-language": "Systemsprache",
+        "language-selector-title": "Sprache auswählen",
+        "hr-or": "ODER",
+        "input-room-id-on-another-device": "Gebe diese Raum ID auf einem anderen Gerät ein",
+        "unpair": "Entkoppeln"
+    },
+    "about": {
+        "tweet_title": "Über PairDrop twittern",
+        "faq_title": "Häufig gestellte Fragen",
+        "close-about_aria-label": "Schließe Über PairDrop",
+        "github_title": "PairDrop auf GitHub",
+        "buy-me-a-coffee_title": "Kauf mir einen Kaffee!",
+        "claim": "Der einfachste Weg Dateien zwischen Geräten zu teilen"
+    },
+    "footer": {
+        "known-as": "Du wirst angezeigt als:",
+        "display-name_title": "Setze einen permanenten Gerätenamen",
+        "discovery-everyone": "Du bist erkennbar von jedem",
+        "on-this-network": "in diesem Netzwerk",
+        "and-by": "und von",
+        "paired-devices": "von gekoppelten Geräten",
```

Comment:
Might be wrong
```suggestion
        "paired-devices": "gekoppelten Geräten",
```

Reply:
Thanks for rechecking! :pray: 

I need to delete the source strings `discovery-everyone` and `and-by`.

The ones that are used with the new version are:

`discovery`

`on-this-network` `paired-devices` `public-room-devices`

Which results in English into this:
![Screenshot from 2023-09-15 13-03-14](https://github.com/schlagmichdoch/PairDrop/assets/52242352/c230a1e4-944c-4b9e-b7ae-8b809c51a59f)

Reply:
> I need to delete the source strings `discovery-everyone` and `and-by`.

Somehow I cannot find them in Weblate anymore. I guess I need to do it manually then.